24lessons/script.js

```diff
@@ -0,0 +1,72 @@
+1
+const firstNumber = +prompt("Please enter a first number:");
+const secondNumber = +prompt("Please enter a second number:");
+
+const minNumber = Math.min(firstNumber, secondNumber);
+const maxNumber = Math.max(firstNumber, secondNumber);
+
+let sumNumber = 0;
+let i = minNumber;
+while (i <= maxNumber) {
+  sumNumber += i;
+  i++;
+}
+alert(`Sum all number = ${sumNumber}`);
+
+//2
+const firstNumber = +prompt("Please enter a first number:");
+const secondNumber = +prompt("Please enter a second number:");
+
+const minNumber = Math.min(firstNumber, secondNumber);
+
+for (; minNumber > 1; minNumber--) {
```

Comment:
тут треба `let`, бо буде помилка
з `const` не можна зробити `minNumber--`
```suggestion
let minNumber = Math.min(firstNumber, secondNumber);

for (; minNumber > 1; minNumber--) {
```